### PR TITLE
docs: fix misspelled words

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Review by borkmann — APPROVED** [review]

**Review by raybejjani — APPROVED** [review]

File: Documentation/configuration/metrics.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ Policy
 
 Events external to Cilium
 -------------------------
-* ``event_ts``: Last timestamp when we received an event. Further labelled by
+* ``event_ts``: Last timestamp when we received an event. Further labeled by
```
Comment (raybejjani) [inline Documentation/configuration/metrics.rst:41]: I'm pretty sure this is a UK vs US thing. I imagine we decided to follow US spellings? If not, we should decide (to use the US ones despite them being wrong ;) )

Comment (aanm) [reply Documentation/configuration/metrics.rst:41]: Yes, US spellings. I've also changed `behaviour` to `behavior` in previous commits

Comment (raybejjani) [reply Documentation/configuration/metrics.rst:41]: that's probably my fault. I'll work on my muscle memory.

**Pull request merged**